cat/CaT-service.yaml

```diff
@@ -2060,6 +2061,255 @@ paths:
             - buyer
       summary: Download a document attached to an event
       
+  /tenders/projects/{proc-id}/events/{event-id}/documents/templates:
+    parameters:
+      - $ref: '#/components/parameters/ProcurementParam'
+      - $ref: '#/components/parameters/EventParam'
+
+    get:
+      description: Get summary information of templates used to create documents attached to an event
+      tags:
+        - Define Procurement Event
+      operationId: getTemplates
+      responses:
+        '200':
+          description: ok
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  allOf:
+                    - $ref: '#/components/schemas/DocumentSummary'
+                    - type: object
+                      description: Metadata relating to a template
+                      properties:
+                        id:
```

Comment:
These properties seem to repeat those in `DocumentSummary` ..?

Reply:
Its just overriding the description to more appropriate text.